scripts/subtitle_search_eval.py

```
@@ -0,0 +1,546 @@
+        season = parsed['season']
+        episode = parsed['episode']
+        folder = p.rsplit('/', 2)[-2] if '/' in p else ''
+
+        # 1) 模拟刮削：用文件名标题去 TMDB 拿英文原名 + imdb_id
+        search_title = build_search_title(title)
+        zh_title = ''
+        original_title = ''
+        imdb_id = ''
+        tmdb_id = ''
+        if not search_title or (not has_english(search_title) and not search_title):
+            # 纯数字/无意义标题，尝试用父目录中文名
```

Ruff (0.16.1)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)